src/pages/filter/ui/modal-edit/components/modal-enum/modal-enum.store.ts

```diff
@@ -128,21 +129,12 @@ class ModalEnumStore {
   }
 
   public get originGroupList(): [string, number][] {
-    const currentStepIndex = activeStepStore.activeStepIndex
-    const currentGroupIndex = dtreeModalStore.groupIndexToChange
+    const selectedGroup = modalEditStore.attributeStatusToChange as StatList
 
-    const currentGroup =
-      dtreeStore.stepData[currentStepIndex].groups[currentGroupIndex]
+    // TODO: getter should not have side effects
```

Comment:
maybe remove side effects if you are here?

Reply:
This is the plan for next refactoring stage... Refactoring for attribute edit panels in refiner and dtree, and enums panel is first in queue )